libcoap/src/context.rs

```diff
@@ -232,7 +233,7 @@ impl CoapContext<'_> {
     pub fn add_endpoint_udp(&mut self, addr: SocketAddr) -> Result<(), EndpointCreationError> {
         // SAFETY: Because we never return an owned reference to the endpoint, it cannot outlive the
         // context it is bound to (i.e. this one).
```

Comment:
Is this comment still needed without the `unsafe` block?

Reply:
You are right. It is redundant. Removed.